src/scancode/cli.py

```diff
@@ -62,6 +62,8 @@
 from commoncode.fileutils import POSIX_PATH_SEP
 from commoncode.timeutils import time2tstamp
 
+from packagedcode import PACKAGE_TYPES
```

Comment:
IMHO we cannot import this globally... Instead make this an option in the packagedcode plugin and not in the main cli.py

Reply:
@pombredanne  Thanks, I have moved the the option in the packagedcode plugin instead.